fixtures/app/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # app
 
+## 3.45.0-pre.1
```

Comment:
hey @amcaplan, should this be `3.45.0-pre.0` or is this correct?

Reply:
Not sure, but it probably doesn't matter much, right?

Reply:
I'm guessing the `pre` numbers are always going to be incremented, while the minor/patch versions will depend on the actual changes. Nothing [here](https://github.com/changesets/changesets/blob/main/docs/prereleases.md) clarifies it for me. We could look into the code if we think this is causing us any problems.